compiler/rustc_hir_typeck/src/fallback.rs

```diff
@@ -9,14 +9,14 @@ use rustc_middle::ty::{self, Ty};
 impl<'tcx> FnCtxt<'_, 'tcx> {
     /// Performs type inference fallback, returning true if any fallback
     /// occurs.
```

Comment:
This comment is now outdated, same with `fallback_if_possible` below.

Reply:
Thanks :smiley: